src/python/espressomd/lbboundaries.py

```diff
@@ -21,10 +29,21 @@ def add(self, *args, **kwargs):
             return lbboundary
 
         def remove(self, lbboundary):
+            """
+            Removes a boundary from the set.
+            
+            Parameters
+            ----------
+            lbboundary : :obj: `LBBoundary`
```

Comment:
is a link generated here? please check, if not you have to provide the full path